src/Symfony/Component/Link/LinkManagerInterface.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Link;
+
+/**
+ * Manages connections between resources according to the W3C specifications.
+ *
+ * @see https://www.w3.org/TR/html5/links.html#links
+ * @see https://www.w3.org/TR/preload/
+ * @see https://www.w3.org/TR/resource-hints/
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+interface LinkManagerInterface
+{
+    /**
+     * Adds an element to the list of resources to preload.
+     *
+     * @param string $uri        The relation URI
+     * @param string $rel        The relation type (e.g. "preload", "prefetch", "prerender" or "dns-prefetch")
```

Comment:
Does it really worth it? Those names make sense in the Twig extension because they are useful for assets. In the component, as pointed out by @evert they can be a lot values.